docs/topics/developer-tools.rst

```diff
@@ -0,0 +1,248 @@
+"text"`` we will see the quote-text we clicked on. The `Inspector` lets you
+copy XPaths to selected elements. Let's try it out: Right-click on the ``span``-
+tag, select ``Copy > XPath`` and paste it in the scrapy shell like so::
+
+    >>> scrapy shell "http://quotes.toscrape.com/"
+    (...)
+    >>> response.xpath('/html/body/div/div[2]/div[1]/div[1]/span[1]/text()').extract()
+    ['"The world as we have created it is a process of our thinking. It cannot be changed without changing our thinking.”]
+
+Adding ``text()`` at the end we are able to extract the first quote with this 
+basic selector. But this XPath is not really that clever. All it does is
+go down a desired path in the source code starting from ``html``. So let's 
+see if we can refine our XPath a bit: 
+
+If we check the `Inspector` again we'll see that directly beneath our 
+expanded ``div``-tag we have eight identical ``div``-tags, each with the 
+same attributes as our first. If we expand any of them, we'll see the same 
+structure as with our first quote: Two ``span``-tags and one ``div``-tag. We can
+expand each ``span``-tag with the ``class="text"`` inside our ``div``-tags and 
+see each quote. With this knowledge we can refine our XPath: Instead of a path
+to follow, we'll simply select all ``span``-tags with the ``class="text"``:: 
+
+    >>> response.xpath('//span[@class="text"]/text()').extract()
+   ['"The world as we have created it is a process of our thinking. It cannot be changed without changing our thinking.”,
+    '“It is our choices, Harry, that show what we truly are, far more than our abilities.”',
+    '“There are only two ways to live your life. One is as though nothing is a miracle. The other is as though everything is a miracle.”',
+    (...)]
+
+And with one simple, cleverer XPath we are able to extract all quotes from 
+the page. We could have constructed a loop over our first XPath to increase 
+the number of the last ``div``, but this would have been unnecessarily 
+complex and by simply constructing an XPath with the ``class="text"`` we 
+were able to extract all quotes in one line. 
+
+The `Inspector` has a lot of other helpful features, such as searching in the 
+source code or directly scrolling to an element you selected. Let's demonstrate
+a use case: 
+
+Say you want to find the ``Next``-button on the page. Type ``Next`` into the 
+search bar on the top right of the `Inspector`. You should get two results. 
+The first is a ``li``-tag with the ``class="text"``, the second the text 
+of an ``a``-tag. Right click on the ``a``-tag and select ``Scroll into View``.
+If you hover over the tag, you'll see the button highlighted. From here
+we could easily create a :ref:`Link Extract <topics-link-extractors>` to 
+follow the pagination. On a simple site such as this, there may not be 
+the need to find an element visually but the ``Scroll into View`` function
+can be quite useful on complex sites. 
```

Comment:
I don't necessarily think that it is inconsistent, since even though the tutorial only focuses on XPaths, I think the focus should be on how to use the Developer Tools, regardless of XPaths or CSS. 

So I added it! :+1: